src/utils/DBsearchDeconvMass_masslist.cpp

```diff
@@ -0,0 +1,512 @@
+// --------------------------------------------------------------------------
```

Comment:
change filename to match class name

Reply:
or what is the difference to src/utils/DBsearchDeconvMass.cpp ?

Reply:
this file should not be included for this pull request. Thank you for pointing it out